addon/components/universal-export/template.hbs

```diff
@@ -1,10 +1,5 @@
-{{#modal-view hidden=hidden closeAction=closeModal container='body'}}
-  <div class="modal-header modal-header__borderless">
-    <button type="button" class="close" data-dismiss="modal" aria-label="Close"
-            id="close-x"><span aria-hidden="true">&times;</span></button>
-    <h4 class="modal-title">{{t "export_influencers.title"}}</h4>
-  </div>
-
+{{#modal-view hidden=hidden closeAction=closeModal centered=true
+  modalHeaderBorderless=true modalTitle=(t "export_influencers.title")}}
```

Comment:
you already know that you're dealing with modal so no need to repeat that on the attribute names. just `borderless` (or `borderlessHeader`) and `title` will suffice imo